test/CQRS/LeanCode.CQRS.AspNetCore.Tests.Integration/ClientsUpdatesTests.cs

```diff
@@ -0,0 +1,118 @@
+using System.Net;
```

Comment:
I'm not sure if such tests should belong to this project - you can unit test handlers without the http fuss. 

Your one has the benefit of testing DI registrations, but I still think this can be done separately.